extract/target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
```

Comment:
✏️ I think we need to update the `.gitignore` to not include files from `target`